Geometry/TrackerGeometryBuilder/test/TrackerDigiGeometryAnalyzer.cc

```diff
@@ -51,6 +51,10 @@
 // class decleration
 //
 
+
+// #define PRINT(X) edm::LogInfo(X)
+#define PRINT(X) std::cout << X << ':'
```

Comment:
it's a /test file. More or less anything goes in this area.
(we are not even running static analyzer on /test subdirectories)